### DISCUSSION TITLE
How to get the name of image from "File" field?

### DISCUSSION THREAD

**Discussion by MarkTanashchuk** [directus/directus#2874] [Archive]
When I add an image to the File field and get the value of this field:
```js
client.getItems("image_field", { single: true }) // JS-SDK
```
I receive a number, not the name of picture.

How can I get name of an image/file entered in the File field?

**Comment by benhaynes** [comment]
You'll need to use the `fields` parameter to get nested relational data, eg: `fields=*.*`

https://docs.directus.io/api/query/fields.html
